test/tls-provider.c

```diff
@@ -7,15 +7,44 @@
  * https://www.openssl.org/source/license.html
  */
 
+#include <stdatomic.h>
```

Comment:
Include `"internal/refcount.h"` instead

Reply:
so changed in https://github.com/openssl/openssl/pull/19312/commits/f716ba4cebb9c0586b746e99554b6f92aada9501